2-networks/main.tf

```diff
@@ -40,15 +40,59 @@ module "shared_vpc_nonprod" {
   project_id           = local.nonprod_host_project_id
   default_region       = var.default_region
   network_name         = "shared-vpc-nonprod"
-  private_service_cidr = "10.1.0.0/16"
+  private_service_cidr = "10.200.0.0/22"
```

Comment:
What's the relationship between the private service CIDR and the subnets?

Reply:
So the private service CIDR is the range which gets allocated for Cloud SQL and other services which use that methodology. When you for example launch your first private MySQL instance, it will carve a /24 out of that allocated block automatically. If you want that range to be routable to on-prem or other networks, it also needs to be on a non-overlapping IP range with other subnets.

Reply:
https://cloud.google.com/vpc/docs/configure-private-services-access